e2e_samples/parking_sensors/devops/templates/jobs/deploy-adf-job.yml

```diff
@@ -36,7 +36,7 @@ jobs:
             location: '$(azureLocation)'
             csmFile: '$(Pipeline.Workspace)/ciartifacts/adf_arm_template/ARMTemplateForFactory.json'
             csmParametersFile: '$(Pipeline.Workspace)/ciartifacts/adf_arm_template/ARMTemplateParametersForFactory.json'
-            overrideParameters: -factoryName "$(adfName)" -P_Ingest_MelbParkingData_properties_0_typeProperties_notebookPath "$(databricksNotebookPath)/02_standardize" -P_Ingest_MelbParkingData_properties_0_typeProperties_0_egg "$(databricksDbfsLibPath)/$(packageWheelName)" -P_Ingest_MelbParkingData_properties_4_typeProperties_0_egg "$(databricksDbfsLibPath)/$(packageWheelName)" -Ls_AdlsGen2_01_properties_typeProperties_url "https://$(datalakeAccountName).dfs.core.windows.net/" -Ls_KeyVault_01_properties_typeProperties_baseUrl "$(kvUrl)" -Ls_Rest_MelParkSensors_01_properties_typeProperties_url "$(apiBaseUrl)" -P_Ingest_MelbParkingData_properties_4_typeProperties_notebookPath "$(databricksNotebookPath)/03_transform" -Ls_AzureDatabricks_01_properties_typeProperties_domain "$(databricksDomain)" -Ls_AzureDatabricks_01_properties_typeProperties_workspaceResourceId "$(databricksWorkspaceResourceId)"
+            overrideParameters: -factoryName "$(adfName)" -P_Ingest_ParkingData_properties_0_typeProperties_notebookPath "$(databricksNotebookPath)/02_standardize" -P_Ingest_ParkingData_properties_0_typeProperties_0_egg "$(databricksDbfsLibPath)/$(packageWheelName)" -P_Ingest_ParkingData_properties_4_typeProperties_0_egg "$(databricksDbfsLibPath)/$(packageWheelName)" -Ls_AdlsGen2_01_properties_typeProperties_url "https://$(datalakeAccountName).dfs.core.windows.net/" -Ls_KeyVault_01_properties_typeProperties_baseUrl "$(kvUrl)" -Ls_Rest_ParkSensors_01_properties_typeProperties_url "$(apiBaseUrl)" -P_Ingest_ParkingData_properties_4_typeProperties_notebookPath "$(databricksNotebookPath)/03_transform" -Ls_AzureDatabricks_01_properties_typeProperties_domain "$(databricksDomain)" -Ls_AzureDatabricks_01_properties_typeProperties_workspaceResourceId "$(databricksWorkspaceResourceId)"
```

Comment:
workspaceResourceId is no longer a property on the LinkedService new definition - I haven't tested yet - but this is going to fail for sure.

Reply:
Then it should be failing today...because all I changed was the references to Melb.

Reply:
The change is not here - is on the databricks linked service. The old version of the  linked service has a property called databricksWorkspaceResourceId - which is no longer the case in the new version and in this code is still being referred here: -Ls_AzureDatabricks_01_properties_typeProperties_workspaceResourceId "$(databricksWorkspaceResourceId - did you ran the CI pipeline with no issues?

Reply:
Ah...I misunderstood. Didn't realize the cascading change from the shared cluster.